### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by GaneshKandu** [pimcore/pimcore#9945] [Q&A / Support]
### Expected behavior

i am not able to install pimcore 10.1.0 using composer

install mentioned on this link
[https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html)

```COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project```

composer downloads v10.0.9 instead of v10.1.0

**it should download v10.1.0 of pimcore/pimcore**

composer,json need to change to
```json
  "require": {
    "pimcore/pimcore": "^10.1"
  },
```

### Actual behavior

**composer downloads v10.0.9 instead of v10.1.0**

### Steps to reproduce

Install Pimcore v10.1.0 using composer 
```COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project```

**Comment by dvesh3** [comment]
@GaneshKandu it works for me:
![image](https://user-images.githubusercontent.com/5137917/128308054-25dbc71e-8912-4793-9fa3-a93c01d2db9c.png)

**Reply by GaneshKandu** [reply]
just tried to install again and its worked !
